### DISCUSSION TITLE
X86 Support

### DISCUSSION THREAD

**Discussion by NexGen-3D-Printing** [HussX/piplay#11] [Ideas]
Well I have gotten this to work on Ubuntu Server, but had to do some manual work, I'll take a look at the script and see what I can do to make it work.

Maybe you can have a script that checks what arch your on and then call the appropriate install script, so you can have one for ARM and one for X86?

The PI 5 is perfectly fine, but will be nice to use some of those now useless Windows Tablets kicking around as Security Camera displays, as these can be found much cheaper than a Pi 5 on its own, I currently use Surface Pro 3's and 4's for this task, but a web browser feed for RTSP is just way to taxing on them, I have tested it working the slowest Tablet I have, a Latitude 7235 2 in 1, and performance was perfect, probably better than the Pi 5.

**Comment by HussX** [comment]
That's easy enough to do, so long as it's Debian/Ubuntu flavored. It's
headless by design and systemctl run so a couple installer tweaks should
make that easily possible. I'll just have to find the difference in apt
packages like you did, or try to make it a bit more native to streamline
the downloads. I actually like relying on apt in this case though.

**Comment by NexGen-3D-Printing** [comment]
Okay, I just installed [Armbian Debian Bookworm](https://www.armbian.com/uefi-x86/) and everything worked perfectly, no issues at all, I just used the server/cli version and your good, git already installed, I run up the same 6 feeds and its far better than the Pi 5 on performance.

The tablet is a really old Getac F110 G2 with a Gen4 Intel CPU, this came with Windows 7 on it, so pretty old, fans going hard, but frame rate looks better than the Pi 5.

Screen isn't the best, but its really just to show you can use old hardware for this task, rather than spend more cash on new gear, I picked up several of these for less than $50 each, they have dual hot swappable batteries, which is perfect for a wall mounted display as you can pull them completely out and it will still work on mains power, the NVME drive has an access door so you can pull that out easy as well, just flash Armbian on it, and put straight back, no messing about, also has ethernet and wifi..

![IMG_20241105_131054_878](https://github.com/user-attachments/assets/b43cd3f4-482c-472c-9121-e5a97138b57d)

**Comment by HussX** [comment]
Love to hear good results. Thanks for trying all this out!
